mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/muleartifact/MuleArtifactMinMuleVersionRuleTest.groovy

```diff
@@ -47,7 +47,6 @@ class MuleArtifactMinMuleVersionRuleTest extends Specification {
         then:
         violations.size() == 1
         violations[0].lineNumber == 2
-        violations[0].fileName.endsWith('pom.xml')
         violations[0].rule.ruleId == MuleArtifactMinMuleVersionRule.RULE_ID
```

Comment:
If this rule referenced the pom wouldn't its line number now be 0 as well like the other pom rules you changed?